src/components/Header/index.tsx

```diff
@@ -43,6 +44,7 @@ const HeaderStyles = styled.header`
     gap: 50px;
 
     @media (max-width: 900px) {
```

Comment:
Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0